tardis/visualization/tools/convergence_plot.py

```diff
@@ -0,0 +1,360 @@
+from collections import defaultdict
+import matplotlib.cm as cm
+import matplotlib.colors as clr
+import plotly.graph_objects as go
+from IPython.display import display
+import matplotlib as mpl
+import ipywidgets as widgets
+from contextlib import suppress
+from traitlets import TraitError
+
+
+def transition_colors(name="jet", iterations=20):
```

Comment:
I don't understand why you're using a default value of iterations, shouldn't it always come from config file?

Reply:
I'm sorry, my bad, the number of iterations always come from the config file, I just forgot to remove `iterations=20`